app/src/main/kotlin/at/bitfire/davdroid/ui/DebugInfoActivity.kt

```diff
@@ -167,12 +195,19 @@ class DebugInfoActivity : AppCompatActivity() {
         fun withLocalResource(dump: String?): IntentBuilder {
             if (dump != null)
                 intent.putExtra(
-                    EXTRA_LOCAL_RESOURCE,
+                    EXTRA_LOCAL_RESOURCE_DUMP,
```

Comment:
Just a personal objection: For some reason I don't like the word "dump" here … (I know we already use it for the database)

Maybe we can use some other word? _string, _summary, something else?